darwin/cli_functions.py

```diff
@@ -335,6 +337,28 @@ def upload_data(
         _error(f"No files found")
 
 
+def dataset_import(dataset_slug, format, files):
+    client = _load_client()
+    try:
+        parser = None
+
+        for (fmt, fmt_parser) in darwin.importer.formats.supported_formats:
+            if fmt != format:
+                continue
+            parser = fmt_parser
```

Comment:
Not sure if that's more readable than
```suggestion
            if fmt == format:
                parser = fmt_parser
```
Also, should we break once we found the parser?